### DISCUSSION TITLE
Separating admin passwd and stream key?

### DISCUSSION THREAD

**Discussion by DidierMalenfant** [owncast/owncast#2319] [Q&A] [answered]
Hey everyone,

Just starting to dip my toes in the owncast ocean and I have a server up and running on Digital Ocean to replace my Mixcloud account.

I have what may be a noob question (sorry if it is), can you separate the admin passwd from the stream key? If I want to share the stream key with other so they can stream on my server but I don't want them to administrate the server.

Thanks for all the hard work on all this btw, this is really really awesome...

**Comment by gabek** [comment, marked as answer]
Filed an issue to work on this in #2320.

**Reply by DidierMalenfant** [reply]
Thank you! Supported the project on OpenCollective. Excited about this project moving forward...

**Comment by gabek** [comment]
This will be released in v0.1.0.
